src/components/CompareResults/beta/ResultsView.tsx

```diff
@@ -84,6 +87,12 @@ function ResultsView(props: ResultsViewProps) {
         view={compareView as View}
       />
       <section className={sectionStyles.container}>
+      <Link href="/" aria-label="link to home">
```

Comment:
I think you need to format this file. If you are using Visual Studio Code the easiest way to do so is to install the extension `Prettier - Code formatter` and use the `Format Document` option.